src/Model.js

```diff
@@ -239,7 +239,9 @@ class Model {
    */
   @computed get isNew () {
     const { id } = this
-    return !!String(id).match(/tmp/)
+    if (!id) return true
+    if (String(id).indexOf('tmp') === -1) return false
+    return true
```

Comment:
I refactored this because `String(undefined) // => 'undefined'`—it still would have made this whole expression false, but I figured I'd try to make it more explicit. Also, changing `match` to `indexOf` since we're just testing for a string literal.